app/models/manageiq/providers/ansible_tower/automation_manager/tower_api.rb

```diff
@@ -1,6 +1,11 @@
 module ManageIQ::Providers::AnsibleTower::AutomationManager::TowerApi
   extend ActiveSupport::Concern
 
+  def self.included(model)
+    model.include SupportsFeatureMixin
+    model.supports :create
```

Comment:
Good with this module defining `supports :create` because the module actually implements `raw_create_in_provider` down on [[line 10]](https://github.com/ManageIQ/manageiq-providers-ansible_tower/pull/274/files#diff-2f371a29d9bf166fc5c677c504445d485bb3cbf53497b922f90f0dfa6a8fdd32R10) (re: https://github.com/ManageIQ/manageiq/pull/21674/files#r796072565)

Reply:
yea, I'm getting cold feet on this one.

think we don't mixin `supports :create` anywhere else. thinking no reason to start now